packages/@uppy/companion/src/server/provider/error.js

```diff
@@ -39,17 +41,38 @@ class ProviderAuthError extends ProviderApiError {
   }
 }
 
+function parseHttpError(err) {
+  if (err?.name === 'HTTPError') {
+    return {
+      statusCode: err.response?.statusCode,
+      body: err.response?.body,
+    }
+  }
+  if (err?.name === 'HttpError') {
```

Comment:
Maybe a comment why we have the same name but just with different casing?

Taking one step back, can't we have a single one? Or rename one to make it more clear?

Reply:
> Maybe a comment why we have the same name but just with different casing?

there's a comment near the declaration of the error class https://github.com/transloadit/uppy/pull/5595/files#diff-0ac4d10dec207b1a38ae56320a1005b2dfc1649da4384191c3cca13b2d3b5d8cR152

basically I don't want to create got's HTTPError because they are a bit too much (they require a deep object structure with a lot of metadata that we don't have).

We could use only HttpError (which is our own custom class), but it's a bit risky rewrite, so I refrain from doing it now.